guide/12-enrich-data-with-thematic-information/part3_enrich_standard_geographies.ipynb

```diff
@@ -0,0 +1,1372 @@
+{
```

Comment:
<p>Values from <strong><em>the </em></strong><code>level_name</code> column can be used as valid input for the <code>enrich</code> <strong><em>method's</em></strong> <code>standard_geography_level</code> parameter.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>